designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/DeploymentComment.scala

```diff
@@ -8,14 +8,15 @@ import pl.touk.nussknacker.ui.api.DeploymentCommentSettings
 import pl.touk.nussknacker.ui.listener.Comment
 import pl.touk.nussknacker.ui.process.repository.DeploymentComment._
 
+// TODO: it does not refer to "deployment" only, rename to ValidatedComment
```

Comment:
I agree

Reply:
Great :) let's do it in another PR (with other renames).